CHANGES.md

```diff
@@ -1,5 +1,152 @@
+  * Added support for dynamically discovering policies for ports that are not documented
+    in a pod's `containerPorts`
+  * Added support for route-based inbound HTTP metrics (`route_group`/`route_kind`/`route_name`)
+
+* Control Plane
+  * Added support for per-route policy by supporting AuthorizationPolicy
+    resources which target HttpRoute resources
+  * Added support for bound service account token volumes for the control plane
+    and injected workloads
+  * Removed kube-system exclusions from watchers to fix service discovery
+    for workloads in the kube-system namespace (thanks @JacobHenner!)
+  * Updated healthcheck to ignore `Terminated` state for pods (thanks
+    @AgrimPrasad!)
+  * Updated the default policy controller log level to `info`; the controller
+    will now emit INFO level logs for some of its dependencies
+  * Added probe authorization by default, allowing clusters that use a
+    default `deny` policy to not explicitly need to authorize probes
+  * Fixed an issue where the proxy-injector would break when using
+    `nodeAffinity` values for the control plane
+  * Fixed an issue where certain control plane components were not restarting
+    as necessary after a trust root rotation
+
+* CLI
+  * Fixed the `linkerd check` command crashing when unexpected pods are found
+    in a Linkerd namespace
+  * Updated the `linkerd authz` command to support AuthorizationPolicy and
+    HttpRoute resources
+  * Updated `linkerd check` to allow RSA signed trust anchors (thanks
+    @danibaeyens!)
+  * `linkerd install --crds` must be run before `linkerd install`
+  * `linkerd upgrade --crds` must be run before `linkerd upgrade`
+  * Fixed invalid yaml syntax in the viz extension's tap-injector template
+    (thanks @wc-s!)
+  * Fixed an issue where the `--default-inbound-policy` setting was not
+    being respected
+  * Added support for AuthorizationPolicy and HttpRoute to `viz authz` command
+  * Added support for AuthorizationPolicy and HttpRoute to `viz stat` command
+  * Added support for policy metadata in linkerd tap
+
+* Helm
+  * Split the `linkerd2` chart into `linkerd-crds` and `linkerd-control-plane`
+  * Charts are now versioned using [SemVer](https://semver.org) independently
+    of Linkerd releases
+  * Added missing port in the Linkerd viz chart documentation (thanks @haswalt!)
+  * Changed the `proxy.await` Helm value so that users can now disable
+    `linkerd-await` on control plane components
+  * Added the `policyController.probeNetworks` Helm value for configuring
+    the networks that probes are expected to be performed from
+
+* Extensions
+  * Added annotations to allow Linkerd extension deployments to be evicted by
+    the autoscaler when necessary
+  * Added a new annotation to configure skipping subnets in the init container
+    (config.linkerd.io/skip-subnets), needed e.g. in Docker-in-Docker workloads
+    (thanks @michaellzc!)
```

Comment:
is this an extension change? is this for CNI? should it go under a CNI bullet?

Reply:
I'd classify it more as an `initContainer` change, not sure if we have a bullet for that, but I'd lob it in with the rest of the init container changes.

Reply:
We don't seem to, no. Moving it to `Proxy`, feel free to move it elsewhere!